### ISSUE TITLE
Change license from GPL to Apache 2.0

### ISSUE THREAD

**Issue by schmmd** [allenai/pdffigures#12]
Hi @chrisc36, we're choosing the Apache 2.0 license as our default license at AI2.  Do you have any concerns changing the license of pdffigures from the GPL to the Apache 2.0 license?

It looks like the only two dependencies are leptonica ([BSD license](http://www.leptonica.com/about-the-license.html)) and poppler (MIT license), both of which are compatible with the Apache 2.0 license.  Please let me know if I'm overlooking any additional dependencies.

**Comment by chrisc36** [comment]
That is correct, those are all the dependencies. I have no concerns about changing the license, in fact I generally prefer Apache 2.0. I (think) that at the time poppler was in the GPL, which is why we used that one initially.

**Comment by schmmd** [comment]
@chrisc36 thanks for the context.  It appears that you are correct and that `poppler` is GPLv2, so this cannot be relicensed as Apache 2.0.
